src/Microsoft.TestPlatform.Common/Utilities/AssemblyResolver.cs

```diff
@@ -85,7 +85,7 @@ internal void AddSearchDirectories(IEnumerable<string> directories)
     /// </returns>
     private Assembly OnResolve(object sender, AssemblyResolveEventArgs args)
     {
-        if (string.IsNullOrEmpty(args?.Name))
+        if (StringUtils.IsNullOrEmpty(args?.Name))
```

Comment:
Cannot use the extension method syntax here because of the `?.`.

Reply:
```suggestion
        if (args is null || args.Name.IsNullOrEmpty())
```

Maybe?

Reply:
It's probably not much better, and maybe even worse because you need to get the condition right.

Reply:
I don't know, both are ok to me. We could maybe go with `arg?.Name is { Length: > 0 }` in this case but for people not used to pattern matching that might be too much.

Reply:
Yeah, way too much. Imho the original static call is okay, and it chains well whe there are more things that can be null in a deeper object structure.

Reply:
Yeah, way too much. Imho the original static call is okay, and it chains well whe there are more things that can be null in a deeper object structure.